### ISSUE TITLE
Difficulties to understand operators bitarray

### ISSUE THREAD

**Issue by Sagebati** [myrrlyn/bitvec#92]
This snippet gives an error using the or bit operator.
```
fn main() {
    let a  = bitarr!();
    let b = bitarr!();
    let c = a | b;
}
```

```
   Compiling test_bitvec v0.1.0 (/home/sam/src/rust/test_bitvec)
error[E0277]: `BitArray<bitvec::order::Lsb0, [usize; 0]>` is not an iterator
 --> src/main.rs:7:15
  |
7 |     let c = a | b;
  |               ^ `BitArray<bitvec::order::Lsb0, [usize; 0]>` is not an iterator
  |
  = help: the trait `Iterator` is not implemented for `BitArray<bitvec::order::Lsb0, [usize; 0]>`
  = note: required because of the requirements on the impl of `IntoIterator` for `BitArray<bitvec::order::Lsb0, [usize; 0]>`
  = note: required because of the requirements on the impl of `BitOrAssign<BitArray<bitvec::order::Lsb0, [usize; 0]>>` for `BitSlice`
  = note: required because of the requirements on the impl of `BitOr` for `BitArray<bitvec::order::Lsb0, [usize; 0]>`
```

I don't understand why ?

**Comment by myrrlyn** [comment]
This will be fixed in the next release.

The reason this does not work currently is that the operators are currently implemented to be generic over any `Iterator<Item = bool>` on the right hand side, and `BitArray` is not yet `IntoIter<Item = bool>` because the implementation in `core` is still unstable so I did not notice it when I was first writing the `BitArray` API.

**Comment by Sagebati** [comment]
Looking forward for your next release, thanks for your time, and awesome crate!

**Comment by myrrlyn** [comment]
This is fixed on current `0.20`, and will be accelerated in the next release per #89.
